app/assets/javascripts/components/mentors/_index.js.jsx

```diff
@@ -0,0 +1,11 @@
+var Index = React.createClass({
+  render() {
```

Comment:
Can we rename this to `MentorIndex`? We'll eventually have more than one index component.